### ISSUE TITLE
bug color 

### ISSUE THREAD

**Issue by gilnett** [ramensoftware/windows-11-taskbar-styling-guide#13]
hello,
the problem is simple the indicator loses its color when starting the PC or when I do an update/download

To solve this problem, 
I just have to deactivate then reactivate the mod.Is there no way to correct this? 
THANKS
![image](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/assets/142233342/5f696567-c3c4-4585-bb46-8ad3923c947e)

![image](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/assets/142233342/36fa012e-3955-434d-81cf-0b83c8f46b1c)

**Comment by gilnett** [comment]
also here are my mod settings

{"controlStyles[0].target":"taskbar:TaskListButton","controlStyles[0].styles[0]":"CornerRadius=8","resourceVariables[0].variableKey":"TaskbarContextMenuMargin","resourceVariables[0].value":"14","controlStyles[1].target":" taskbar:TaskListLabeledButtonPanel@RunningIndicatorStates > Rectangle#RunningIndicator","controlStyles[1].styles[0]":"Fill=#FFED7014","controlStyles[1].styles[1]":"Height=3","controlStyles[1].styles[2]":"Width=20","controlStyles[1].styles[3]":"Fill@ActiveRunningIndicator=Red","controlStyles[1].styles[4]":"Width@ActiveRunningIndicator=25","controlStyles[2].styles[0]":"Fill=Transparent","controlStyles[2].target":"Rectangle#BackgroundFill","controlStyles[3].target":"Rectangle#BackgroundStroke","controlStyles[3].styles[0]":"Visibility=Collapsed"}

**Comment by m417z** [comment]
Any chance you're using TranslucentTB? There's a known incompatibility with it, see https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/9#issuecomment-1896063781

**Comment by gilnett** [comment]
No, I don't use transluscenTB. On the other hand, I use Itop easy desktop which offers to modify the task bar, but this option is deactivated.
maybe it doesn't change well because of a verification request on the Itop easy desktop side (to see if it's activated or not)


also this has nothing to do with it but I wondered if there was a way to put the application image in the background. (some of my applications hide the indicator)

**Comment by m417z** [comment]
I'm not familiar with Itop easy desktop, but maybe, you can turn it off completely and test it.

> I wondered if there was a way to put the application image in the background

I'm not familiar with a way to do that. As an alternative, you can move it to a lower position:
https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/12#issuecomment-1906680866

**Comment by gilnett** [comment]
ok thank you for your help so I made it simpler: 
I added itop easy desktop to the axclusion list. of the mod Apparently it works well
this also allowed me to keep the other options of itop easy desktop without causing a bug.

**Comment by m417z** [comment]
> the axclusion list. of the mod

Which list are you referring to? The one in the advanced tab? It should have no effect since the mod only targets explorer.exe anyway.

**Comment by gilnett** [comment]
In reality, it doesn't work. It seems like it was a fortunate coincidence.

**Comment by sylveon** [comment]
iTop Easy Desktop is stealing TranslucentTB code without honoring the GPL, that's why it is causing issues:
![image](https://github.com/user-attachments/assets/ffb5a64f-5150-49a3-b590-1b08c9ee9a43)

**Comment by gilnett** [comment]
I didn't know that, but I admit it's shameful. I guess I'll go back to the apps I used before, namely Windhawk and TranslucentTB. As for organizing the folders, we'll go back to the classic desktop.

**Comment by bbmaster123** [comment]
@gilnett 
sorry I must have missed this issue

to me, it sounds like the progress bar resets the running indicator styles, making you have to reload the mod. 

If removing iTop doesn't fix the issue, could you try adding the same running indicator styles to these 4 targets?

Border#ProgressBarRoot
Rectangle#DeterminateProgressBarIndicator
Rectangle#IndeterminateProgressBarIndicator
Rectangle#IndeterminateProgressBarIndicator2

those affect the download progress bar and calculating bar (for example, when copying lots of files and it takes a bit before the copy starts)

**Comment by gilnett** [comment]
I have set the 4 targets and all the styles. I'll test and get back to you in a while once I have checked everything.

**Comment by bbmaster123** [comment]
co-op mod? what is that? No, I'm not so skilled to create a whole mod, I just help out with styler mod themes :)
> I'll test and get back to you in a while once I have checked everything.

Sounds good! :)

**Comment by gilnett** [comment]
Unfortunately, it's not working. But anyway, thanks

I was just asking about the co-op mod :)

**Comment by bbmaster123** [comment]
I'm not sure what mod that is, I've never heard of a co-op mod for windhawk... are you sure that's what its called?
Anyway, one other thing you can try is to add a few seconds delay to windhawk's startup. You can do this by going to settings at the top right of Windhawk, more advanced settings, and put in a value in miliseconds. 
By the way, does it do this with any of the other styles too? like does it also lose its `CornerRadius=8`?

**Comment by gilnett** [comment]
No, you're mistaken. I wasn't saying there was a co-op mod, I was just saying that I thought you made the mod with @m417z .
I will try delay and built in themes work fine .
However, even in the themes, the "Rectangle#RunningIndicator" doesn't work.

**Comment by bbmaster123** [comment]
ahhh sorry about that! I understand you now, you're saying you had thought we had both worked on the styler mod together! I just provide feedback sometimes and try to help out where I can. sometimes language barriers be like that haha :)

okay let me know if the delay helps!

**Comment by m417z** [comment]
Windhawk doesn't provide a way to delay mod loading. @bbmaster123 I assume you're referring to this option:

![image](https://github.com/user-attachments/assets/5000de34-cf17-4ed9-a0e7-6f0d6bd88148)

The option controls the amount of time to wait before showing the progress dialog:

![image](https://github.com/user-attachments/assets/2f6fc5e6-c356-447b-8266-9365fa908a22)

**Comment by bbmaster123** [comment]
yea that was what I was referring to, I guess that won't do anything...

**Comment by gilnett** [comment]
Ah, so I didn't read it wrong :) Well, it's not a big deal, thanks anyway. Plus, now I'm using the themes.
